ocrd_monitor/ocrdmonitor/server/templates/workspace.html.j2

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html.j2' %}
+
+{% block headline %}
+{% block title %}OCRD Browser{% endblock %}
+{% endblock %}
+{% block content %}
+<style>
+    iframe {
```

Comment:
JS functionalities to wait for OCR browser is not needed anymore?

Reply:
I did encounter some problems while launching the browser. I'll implement some waiting behavior.
However, I don't think that's a blocker for merging as a page refresh will do the trick.

Reply:
I've just implemented a simple waiting feature that should ensure that the user never has to see an error due to the browser not being ready yet.